### PR TITLE
Corrected errors in Readme.md

### PR DESCRIPTION
**Pull request by Parth4git**
Corrected the mistakes in readme.md to increase readability

### CODE REVIEW

**Comment by Parth4git** [comment]
@Vaibhavs10 @ariG23498 Can you Please review my PR

File: .github/README.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ print(f"Generation: {generation[0]['generated_text']}")
...
 #   {'role': 'system', 'content': 'You are a helpful assistant, that responds as a pirate.'},
 #   {'role': 'user', 'content': "What's Deep Learning?"},
-#   {'role': 'assistant', 'content': "Yer lookin' fer a treasure trove o'
-#             knowledge on Deep Learnin', eh? Alright then, listen close and
-#             I'll tell ye about it.\n\nDeep Learnin' be a type o' machine
-#             learnin' that uses neural networks"}
+#   {'role': 'assistant', 'content': "You looking fer a treasure trove of
+#             knowledge on Deep Learning, eh? Alright then, listen close and
+#             I'll tell you about it.\n\n Deep Learnin' be a type of machine
+#             learning that uses neural networks"}
```
Comment (ariG23498) [inline .github/README.md:63]: This is not a mistake. We had asked the model to respond like a pirate.

File: .github/README.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Working on the 🐘 big Llama 3.1 405B model:
 
 ## Model Fine Tuning:
 
-It is often not enough to run inference on the model. 
+It is often not enough to run inferences on the model.
```
Comment (ariG23498) [inline .github/README.md:101]: Let's revert back to inference as I do not seem to think it is a grammatical error.

File: .github/README.md [hunk 3]
```diff
@@ -154,8 +154,8 @@ Seeking an entry-level RAG pipeline? This notebook guides you through building a
 * [Simple RAG Pipeline](../llama_rag/llama_rag_pipeline.ipynb)
 
 
-## Text Generation Inference (TGI) & API Inference with Llama Models
-Text Generation Inference (TGI) framework enables efficient and  scalable deployment of Llama models. In this notebook we'll learn how to integrate TGI for fast text generation and to consume already deployed Llama models via Inference API:
+## Text Generation Inference (TGI) and API Inference with Llama Models
```
Comment (ariG23498) [inline .github/README.md:157]: Let's revert back to `&`

**Review by ariG23498 — CHANGES_REQUESTED** [review]
Left some minor nits.

**Comment by Parth4git** [comment]
Now , Please review the changes you mentioned are implemented. If everything in fine , Please merge the PR
@ariG23498 @pcuenca

**Comment by Parth4git** [comment]
@ariG23498 ,Please take a look to PR

**Comment by pcuenca** [comment]
Hi @Parth4git, the people that need to review this PR are already receiving notifications, there's no need to ping them repeatedly about it. If you want to build lasting relationships with others in the community, I'd recommend you try to be patient and respectful of their time and availability.

**Comment by Parth4git** [comment]
@pcuenca , Yes I respect their time and availability. But just wanted to know about any update PR status , I have participated in Devfest.ai event and only 3 days are left for event to end. If my PR get merged I will get points . That why I wanted to know about status of my PR

**Review by ariG23498 — APPROVED** [review]
Looks good to me.

**Comment by Parth4git** [comment]
thank you

**Pull request merged**